releng/release-go-changelog-update

```diff
@@ -114,6 +114,36 @@ def main(next_ver: str, today: datetime.date, quiet: bool=False, commit: bool =
 
         git_add("CHANGELOG.md")
 
+    with check(f"Updating releaseNotes.yml with date for {next_ver}..."):
+        state = 0
+
+        for line in fileinput.FileInput("docs/releaseNotes.yml", inplace=True):
+            if state == 0:
+                if line.strip() == f"- version: {next_ver}":
+                    state = 1
+            elif state == 1:
+                m = re.match(r"(\s+date:)\s+.*$", line)
+
+                if m:
```

Comment:
We have the walrus operator now (since Python 3.8)
```python
if m := re.match(r"(\s+date:)\s+.*$", line):
```
:)

Reply:
So in general I'm still not gonna be loving the walrus operator... but this _is_ the place to use it, really...